isic_archive/tasks/__init__.py

```diff
@@ -0,0 +1,9 @@
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
Consider promoting this to the top level of `isic_archive` (not just `tasks`). Also consider making an `isic_archive.settings` (or similar) where this lives.